src/oc/x.c

```diff
@@ -12,9 +13,17 @@
 
 #if NRNOC_X11
 
+#if defined(IVX11_DYNAM)
+#include <IV-X11/ivx11_declare.h>
+#include <IV-X11/ivx11_redef.h>
+extern int hoc_usegui;
+#define ifnox_return if(!hoc_usegui) { return; }
```

Comment:
nitpick : may be extra underscore `if_nox_return` read better? (or `return_if_no_x`)